contracts/src/v0.8/functions/dev/1_0_0/FunctionsRouter.sol

```diff
@@ -54,15 +54,7 @@ contract FunctionsRouter is RouterBase, IFunctionsRouter, FunctionsSubscriptions
   // ================================================================
   // |                    Configuration state                       |
   // ================================================================
-  struct Config {
-    // Flat fee (in Juels of LINK) that will be paid to the Router owner for operation of the network
-    uint96 adminFee;
-    // The function selector that is used when calling back to the Client contract
-    bytes4 handleOracleFulfillmentSelector;
-    // List of max callback gas limits used by flag with GAS_FLAG_INDEX
-    uint32[] maxCallbackGasLimits;
-  }
-  Config private s_config;
+  SubscriptionConfig private s_config;
```

Comment:
So it is a subscription config now but it's still stored in the Router? Doesn't sound right.

Reply:
I agree with @bolekk. This shouldn't be changed.